examples/app.py

```diff
@@ -25,6 +25,8 @@
 
 u"""Minimal Flask application example for development.
 
+Run Redis server.
```

Comment:
There is already mention about the redis later in the documentation, so maybe we shouldn't duplicate it: https://github.com/hachreak/invenio-accounts/blob/b2620e66978ebad95da50f01f430efaffdfe793f/examples/app.py#L42